src/applications/simple-forms/20-10207/pages/idInfoThirdPartyVeteran.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import {
+  fullNameNoSuffixUI,
+  fullNameNoMiddleNoSuffixSchema,
+  radioUI,
+  radioSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+import {
+  THIRD_PARTY_TYPES,
+  THIRD_PARTY_TYPE_VETERAN_LABELS,
+  ADDITIONAL_INFO_THIRD_PARTY_TYPE,
+} from '../config/constants';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    'ui:title': <h3 className="vads-u-margin-y--0">Your name</h3>,
+    thirdPartyFullName: fullNameNoSuffixUI(null, { omitMiddle: true }),
```

Comment:
left comment below about maybe using a dedicated pattern for this

Reply:
Now using ...titleUI(...)